cmd/memparse.go

```diff
@@ -273,3 +293,48 @@ func generateHexAndAscii(data []byte) (string, string) {
 
 	return hex, ascii
 }
+
+func printMemorySearchResult(task internal.Task) error {
+	memReader, err := crit.NewMemoryReader(
+		filepath.Join(task.OutputDir, metadata.CheckpointDirectory),
+		*pID, pageSize,
+	)
+	if err != nil {
+		return fmt.Errorf("failed to create memory reader: %w", err)
+	}
+
+	if err := internal.UntarFiles(
+		task.CheckpointFilePath, task.OutputDir,
+		[]string{filepath.Join(metadata.CheckpointDirectory, fmt.Sprintf("pages-%d.img", memReader.GetPagesID()))},
+	); err != nil {
+		return fmt.Errorf("failed to extract pages file: %w", err)
+	}
+
+	results, err := memReader.SearchPattern(*searchPattern, *searchContext)
+	if err != nil {
+		return fmt.Errorf("failed to search pattern in memory: %w", err)
+	}
+
+	if len(results) == 0 {
+		fmt.Printf("\nNothing found for pattern %s in memory pages of process %d", *searchPattern, *pID)
```

Comment:
```suggestion
		fmt.Printf("\nNo matches for pattern %s in memory pages of process %d", *searchPattern, *pID)
```
I think this reads a bit better than "nothing found"